flatdata-rs/lib/src/generator.rs

```diff
@@ -1,6 +1,7 @@
 use std::{
-    io::Write,
-    path::{Path, PathBuf},
+    env,
+    path::PathBuf,
+    process::{Command, Stdio},
 };
 
 /// A helper function wrapping the flatdata generator.
```

Comment:
I think we should add in the documentation that the files are generated in the OUT_DIR.